SECURITY.md

```diff
@@ -0,0 +1 @@
+will@grid.ai
```

Comment:
is there a general support email for the lightning project that can be added as well?

Reply:
I think that it shall go to @williamFalcon and he can forward action to the team, but the info can be sensitive so rather not share in general email...

Reply:
I think that the developers should be included directly as potential security fixes would have to be fixed asap.

Reply:
```suggestion
Security@grid.ai
```